### PR TITLE
ValueTuple: Lowering framework dependency to netstandard1.0

### PR DESCRIPTION
**Pull request by jcouv**
Tomas pointed out that the dependency on netstandard1.1 is causing him some trouble and it is un-necessary. Changing to 1.0.
Note: I picked this up from `System.Buffers`. You may want to consider a similar change there.

@stephentoub @tmat for review.

### CODE REVIEW

**Comment by stephentoub** [comment]
cc: @ericstj

File: src/System.ValueTuple/src/project.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "System.Runtime": "4.0.0"
   },
   "frameworks": {
-    "netstandard1.1": {
+    "netstandard1.0": {
       "imports": [
         "dotnet5.2"
```
Comment (ericstj) [inline src/System.ValueTuple/src/project.json]: Import should be 5.1, otherwise you might accidentally use API in 1.1.

Comment (jcouv) [inline src/System.ValueTuple/src/project.json]: Thanks a lot. I'll change immediately.

**Comment by jcouv** [comment]
I'm investigating the build error: `12:11:30 D:\j\workspace\windows_nt_re---37265eab\Tools\Packaging.targets(951,5): error : System.ValueTuple should not be supported on WindowsPhone,Version=v8.0 but has both compile and runtime assets. [D:\j\workspace\windows_nt_re---37265eab\src\System.ValueTuple\pkg\System.ValueTuple.pkgproj]`

I feel like this problem is probably related to the [netstandard map](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#mapping-the-net-platform-standard-to-platforms), but that map doesn't say anything about Windows Phone 8.0...

@stephentoub @ericstj Could you shed some light?

**Comment by ericstj** [comment]
You need to update the PKGProj to declare where the package is supported.  https://github.com/dotnet/corefx/blob/master/src/System.ValueTuple/pkg/System.ValueTuple.pkgproj#L7

**Comment by jcouv** [comment]
@dotnet-bot retest Innerloop OSX Debug Build and Test please
// Previous failure: http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/master/job/osx_debug_prtest/2154/
// Retest reason:

**Comment by jcouv** [comment]
@dotnet-bot retest Innerloop OSX Debug Build and Test please
// Previous failure: http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/master/job/osx_debug_prtest/2154/
// Retest reason: remoting error

**Comment by jcouv** [comment]
Thanks Eric for the tip.
@ericstj @stephentoub Please review the change and merge when you're ok.

**Comment by ericstj** [comment]
LGTM

**Comment by jcouv** [comment]
Thanks!

**Pull request merged**